index.html

```diff
@@ -0,0 +1,253 @@
+              <h5 class="col-sm-5 col-12 col-md-12 pr-0 pl-0 orange section-subtitle">
+                Sound Test
+              </h5>
+              <p class="custom_p_tag col-sm-5 col-12 col-md-12 pb-4 pr-0 pl-0 text-white">
+                This is the time when everyone will show the car's whine and sutututu.
+              </p>
+            </article>
+          </li>
+          <li class="col-12 col-md-3 pr-2 pl-0 mr-1">
+            <article class="program-block text-center row mb-5 p-4 bg-dark-light-custom">
+              <div class="col-sm-2 col-12 col-md-12 pr-0 pl-0">
+               <img src="assets/images/drift-2.png" alt="lecture"/>
+              </div>
+              <h5 class="col-sm-5 col-12 col-md-12 pr-0 pl-0 orange section-subtitle">
+                Drifting
+              </h5>
+              <p class="custom_p_tag col-sm-5 col-12 col-md-12 pb-4 pr-0 pl-0 text-white">
+                We will move our location so the cars can drift around, doing donuts and everything else.
+              </p>
+            </article>
+          </li>
+          <li class="col-12 col-md-3 pr-2 pl-0 mr-1">
+            <article class="program-block text-center row mb-5 p-4 bg-dark-light-custom">
+              <div class="col-sm-2 col-12 col-md-12 pr-0 pl-0">
+               <img src="assets/images/approve-2.png" alt="lecture"/>
+              </div>
+              <h5 class="col-sm-5 col-12 col-md-12 pr-0 pl-0 orange section-subtitle">
+                Contest
+              </h5>
+              <p class="custom_p_tag col-sm-5 col-12 col-md-12 pb-4 pr-0 pl-0 text-white">
+                We will gather votes, and the person that has the most, will get an Akarprovic Exhaust for his car.
+              </p>
+            </article>
+          </li>
+          <li class="col-12 col-md-3 pr-3 pl-0 mr-1">
+            <article class="program-block text-center row mb-5 p-4 bg-dark-light-custom">
+              <div class="col-sm-2 col-12 col-md-12 pr-0 pl-0">
+               <img src="assets/images/winner-2.png" alt="lecture"/>
+              </div>
+              <h5 class="col-sm-5 col-12 col-md-12 pr-0 pl-0 orange section-subtitle">
+                The Reveal
+              </h5>
+              <p class="custom_p_tag col-sm-5 col-12 col-md-12 pb-4 pr-0 pl-0 text-white">
+                We will reveal the winner. The other contenstants will get something as well, but it's a surprize.
+              </p>
+            </article>
+          </li>
+        </ul>
+        <div class="row mb-5">
+          <div class="col-12 text-center pb-4 d-none d-md-block">
+            <a href="#" class="text-white">
+              SEE THE WHOLE PROGRAM
+            </a>
+          </div>
+          <div class="col-12 text-center d-md-none d-xs-block">
+            <a href="#" class="text-white cta btn rounded-0 bg-orange mb-3 pr-5 pl-5 pt-4 pb-4">
+              Join Crimson Dawn 22
+            </a>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] • Great program section display. 👏 Let's make it even greater by following the margins, padding, and widths of the original design.
- Your design:
<img width="944" alt="image" src="https://user-images.githubusercontent.com/77190203/157695786-140a24f9-a6d5-4573-b88c-5127a1c9d2a1.png">

- Original design:
<img width="625" alt="image" src="https://user-images.githubusercontent.com/77190203/157696191-c284a77e-009b-47af-b676-8200601f2681.png">